src/app/feedback/feedback-widget/feedback-widget.component.ts

```diff
@@ -26,9 +26,20 @@ export class FeedbackWidgetComponent implements OnInit {
   @HostListener('window:mousedown')
   handleDialogClose() {
     // TODO: Move out to a directive
-    const belongsToPopup = event['path'].some(item =>
-      item.className && item.className.includes('feedback-container')
-    );
+    function findMatchingDOMAncestor(element) {
+      while (element.parentNode) {
+        if (element.className && element.className.indexOf('feedback-container') >= 0) {
```

Comment:
Automatic formatting? I ran the shortcut we installed but didn't see any changes. If you tell me what's wrong with the format I can manually fix it?